utils/test_results.py

```diff
@@ -19,14 +19,21 @@
 from shared.django_apps.reports.models import (
     DailyTestRollup,
     Flake,
-    TestInstance,
 )
 
 thirty_days_ago = dt.date.today() - dt.timedelta(days=30)
 
 SLOW_TEST_PERCENTILE = 95
 
 
+def slow_test_threshold(num_tests: int):
```

Comment:
this can be simplified to 

percentile = (100-95)/100

slow_tests_to_return = floor(percentile * total_tests)
return max(slow_tests_to_return,1)